app/models/manageiq/providers/base_manager.rb

```diff
@@ -2,6 +2,8 @@ module ManageIQ::Providers
   class BaseManager < ExtManagementSystem
     require_nested :Refresher
 
+    validate :require_credentials
```

Comment:
Checked this against the latest master (with delegation to endpoints) and it seems fine for now.